configurator/src/main/java/com/devonfw/ide/configurator/Configurator.java

```diff
@@ -100,10 +100,38 @@ private void saveChangesInWorkspace(boolean saveNewProperties) {
   private VariableResolver createResolver(File variablesFile) {
 
     Properties variables = PropertiesMerger.loadIfExists(variablesFile);
-    variables.put(CLIENT_ENV_HOME_VARIABLE, CURRENT_WORKING_DIRECTORY);
```

Comment:
this was the only variable that could be resolved. However, the settings also contain variables like `SETTINGS_PATH` that need to be resolved. I added all variables I thought are required and make sense.
BTW: For most projects I tried, the `variablesFile` (replacement-patterns.properties) does not exist. Maybe it would be better to configure the path to it in the `devon-ide` scripts itself rather than in the settings (`ide-properties.bat` - that also lacks multi-platform support).